tests/Fixer/Comment/NoEmptyCommentFixerTest.php

```diff
@@ -66,10 +66,12 @@ public function provideFixCases()
             ),
             array(
                 '<?php
+                echo 2;
```

Comment:
These shouldn't, I've added the missing case just to make sure the fixer works on 2.x.